src/Core/Customer/Infrastructure/Repository/MongoCustomerRepository.php

```diff
@@ -14,6 +14,7 @@
 final class MongoCustomerRepository extends BaseRepository implements
     CustomerRepositoryInterface
 {
+    /** @psalm-suppress PossiblyUnusedMethod */
```

Comment:
We are not using the suppress annotations, please check if this code is unused and if so, remove it